tests/SampleApps/nodejs/linxnodeexpress-appinsights/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "A sample Node.js app using Express 4 and application insights",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "cross-env GLOBAL_PATH=/usr/local/lib/node_modules/ node server.js"
```

Comment:
why?? what happens when user doesn't provvide cross-env GLOBAL_PATH

Reply:
Removed cross-env and the env var